src/multi-language.js

```diff
@@ -52,8 +55,18 @@ function addFieldsToSchema(schema, { fieldset }) {
 
     return {
       ...result,
-      language: pluginConfig.defaultLanguage,
+      language: getParentRefLanguageHack() ?? pluginConfig.defaultLanguage,
       translationId: uuid.v4()
     }
   }
 }
+
+function getParentRefLanguageHack() {
+  const segments = decodeURIComponent(window.location.pathname).split(';')
+  const currentSegment = segments.slice(-1).shift()
+  const [parentId] = segments.slice(-2).shift()?.split(',')
```

Comment:
Je kan nog even kijken, maar volgens mij zit deze info ook in 1 van de hooks (misschien die van de router).

Reply:
Hm, ja sowieso, alleen kan ik hier geen hooks gebruiken omdat het geen custom hook of react component is. Ik zal nog even kijken of ik wellicht Sanity spul kan gebruiken om de url te parsen

Reply:
No luck. Wellicht dat jij snapt hoe `getStateFromPath` werkt? Daar moet ik een root node hebben, maar het is me onduidelijk wat dat is of waar ik die vandaag kan halen. Ik weet niet of dat iets is, laten we daar in dat geval later een patch release voor maken.